test/TestUtilities/Test.Utility/Signing/SigningTestUtility.cs

```diff
@@ -662,20 +662,7 @@ public static void VerifyByteArrays(byte[] expected, byte[] actual)
 
         public static void AssertOfflineRevocation(IEnumerable<ILogMessage> issues, LogLevel logLevel)
         {
-            string offlineRevocation;
-
-            if (RuntimeEnvironmentHelper.IsWindows)
-            {
-                offlineRevocation = "The revocation function was unable to check revocation because the revocation server was offline";
-            }
-            else if (RuntimeEnvironmentHelper.IsMacOSX)
-            {
-                offlineRevocation = "An incomplete certificate revocation check occurred.";
-            }
-            else
-            {
-                offlineRevocation = "unable to get certificate CRL";
-            }
+            string offlineRevocation = X509ChainStatusFlags.OfflineRevocation.ToString();
```

Comment:
This is awesome! I didn't know this existed!

Reply:
Thanks! :)